pyqg/tests/test_parameterizations.py

```diff
@@ -49,5 +49,16 @@ def test_addition_and_scaling(model, rtol=1e-11):
     with pytest.raises(AssertionError):
         back + smag
 
+@pytest.fixture(params=[QG])
+def model(request):
+    klass = request.param()
+    model = klass()
+    model._step_forward()
+    return model
```

Comment:
Is there a reason you need to redefine `model` just with `QG`? If it works with other models, then you could just remove this and just include the test you have below.

Reply:
I've never used the parametrization with the other models...

Reply:
The tests passed locally for my most recent commit for all models.